combat-trainer.lic

```diff
@@ -904,6 +905,24 @@ class SafetyProcess
 
   private
 
+  def recover_items(game_state)
+    return unless Flags['ct-germshieldlost']
+    get_item_wear_item(Flags['ct-germshieldlost'].first)
```

Comment:
Does this need `game_state` passed to it?

Reply:
```
[combat-trainer]>analyze
A misshapen germish'din growls angrily as it reaches out for you!
You try to dodge, but the misshapen germish'din manages to grab your sipar!
It jerks the round sipar out of your hands and slings it aside!
[You're off balance with opponent in better position.]
>
You fail to find any holes in the germish'din's defenses.
>
[combat-trainer]>analyze
Your analysis reveals a massive opening already being exploited in a misshapen germish'din's defenses.
A chance for a stun can be inflicted by landing a thrust, a draw and a jab.
Roundtime: 1 sec.
>
[combat-trainer]>get sipar
You pick up the sipar lying at your feet.
>
[combat-trainer]>wear my sipar
[You're off balance and in better position.]
>
[You're off balance and in better position.]
>
You slide your left arm through the straps on your round sipar and flex, gaining a feel for the sipar's weight.
>
[combat-trainer]>weave
>
fight.
[You're slightly off balance and in good position.]
Roundtime: 4 sec.
```

```[combat-trainer]>bob
[You're adeptly balanced and in dominating position.]
>
A misshapen germish'din growls angrily as it reaches out for you!
You try to dodge, but the misshapen germish'din manages to grab your sipar!
It jerks the round sipar out of your hands and slings it aside!
[You're nimbly balanced and overwhelming your opponent.]
>
You bob suddenly, lowering yourself into a smaller target.
[You're adeptly balanced and overwhelming opponent.]
Roundtime: 3 sec.
>
[combat-trainer]>get sipar
...wait 1 seconds.
>
[combat-trainer]>get sipar
...wait 1 seconds.
>
[combat-trainer]>get sipar
[You're adeptly balanced and in dominating position.]
>
...wait 1 seconds.
>
A misshapen germish'din shambles in, growling angrily.
>
[combat-trainer]>get sipar
>
[You're nimbly balanced and in very strong position.]
>
You pick up the sipar lying at your feet.
>
[combat-trainer]>wear my sipar
You slide your left arm through the straps on your round sipar and flex, gaining a feel for the sipar's weight.
```